README.md

```diff
@@ -330,3 +330,19 @@ var objectType = new minim.ObjectType()
   .set('email', 'john@example.com')
   .set('id', 4)
 ```
+
+### Errors
+
+Minim has its own error type that is passed down through the call chain. This allows clients to write chains that may break but do not throw JavaScript errors.
+
+```javascript
+var objectExample = {
+  foo: {
+    bar: 'value 1'
+  }
+};
+
+var objectType = minim.ObjectType(objectExample);
+var value1 = objectType.get('foo').get('bar'); // value of foo.bar
+var notFound = objectType.get('baz').get('foo') // Returns error object
```

Comment:
So, I am still uncomfortable that I chain something and anywhere in that chain can be an error element and nothing tells me it is there. This feels very uncomfortable to me. It seems a greater evil to hide something wrong than to fail hard and you have to trap it or introduce a try method if you are concerned like:

```coffeescript
object.try('get', 'baz').try('get', 'foo') 
```
or something that returns undefined if you want to gracefully do something. Maybe I a missing something, no pun intended.